src/components/CategoryFilter.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const categories = ['All', 'Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+const CategoryFilter = (props) => {
+  const { target, handleFilter } = props;
+
+  return (
+    <select name="target" value={target} onChange={handleFilter}>
+      {categories.map((category) => (
+        <option key={category} value={category}>
+          {category}
+        </option>
+      ))}
+    </select>
+  );
+};
+
+CategoryFilter.propTypes = {
+  target: PropTypes.string.isRequired,
+  handleFilter: PropTypes.func.isRequired,
+};
+
```

Comment:
[OPTIONAL] 

- [ ] Consider tidying up the `ID` field of the application. The hexadecimal unique number generated for each book does not look tidy. You can explore other means of generating random numbers.
- [ ] Please update the live version of your application with recent features you have added.